src/containers/Notifications.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { prefetch } from 'react-fetcher'
+import { fetchActivity, markActivityRead, markAllActivitiesRead, navigate, thank } from '../actions'
+import { values, sortBy, isEmpty, contains } from 'lodash'
+import cx from 'classnames'
+import ScrollListener from '../components/ScrollListener'
+import A from '../components/A'
+import truncate from 'html-truncate'
+import { present, humanDate } from '../util/text'
+const { array, bool, func, number, object } = React.PropTypes
+
+@prefetch(({ dispatch }) => dispatch(fetchActivity(20, 0)))
+@connect(state => ({
+  activities: sortBy(values(state.activities), ['created_at']).reverse(),
```

Comment:
wait, that's not going to be sufficient, because you're not going to get the new ones. crap, this is a bug with all of our pagination. the way i've solved this in the past (don't know how i overlooked this here) is to pass the id or timestamp of the item at the front, and then add a clause on the backend to include only items with id/timestamp equal to or less than that one